static/app/views/ddm/customMetricsEventData.tsx

```diff
@@ -0,0 +1,206 @@
+                            op: getDefaultMetricOp(mri),
+                          },
+                        ],
+                      })}
+                    >
+                      {t('Open in Metrics')}
+                    </LinkButton>
+                  ),
+                },
+                {
+                  key: 'stats',
+                  subject: t('Stats'),
+                  value: <MetricStats item={item} mri={mri} />,
+                },
+                item.tags && Object.keys(item.tags).length > 0
+                  ? {
+                      key: 'tags',
+                      subject: t('Tags'),
+                      value: (
+                        <Tags tags={item.tags} organization={organization} mri={mri} />
+                      ),
+                    }
+                  : null,
+              ].filter((row): row is Exclude<typeof row, null> => Boolean(row))}
+            />
+          </Fragment>
+        );
+      })}
+    </EventDataSection>
+  );
+}
+
+function MetricStats({mri, item}: {item: MetricsSummaryItem; mri: MRI}) {
+  const parsedMRI = parseMRI(mri);
+  const unit = parsedMRI?.unit ?? 'none';
+  const type = parsedMRI?.type ?? 'c';
+
+  const typeLine = t(`Type: %s`, getReadableMetricType(type));
+  // We use formatMetricUsingUnit with unit 'none' to ensure uniform number formatting
+  const countLine = t(`Count: %s`, formatMetricUsingUnit(item.count, 'none'));
+
+  const baseStats = (
+    <Fragment>
+      {typeLine}
+      <br />
+      {countLine}
+    </Fragment>
+  );
+
+  // For counters the other stats offer little value, so we only show type and count
+  if (type === 'c' || !item.count) {
+    return <pre>{baseStats}</pre>;
+  }
+
+  // If there is only one value, min, max, avg and sum are all the same
+  if (item.count <= 1) {
+    return (
+      <pre>
+        {baseStats}
+        <br />
+        {t('Value: %s', formatMetricUsingUnit(item.sum, unit))}
+      </pre>
+    );
+  }
+
+  return (
+    <pre>
+      {baseStats}
+      <br />
+      {t('Sum: %s', formatMetricUsingUnit(item.sum, unit))}
+      <br />
+      {t('Min: %s', formatMetricUsingUnit(item.min, unit))}
+      <br />
+      {t('Max: %s', formatMetricUsingUnit(item.max, unit))}
+      <br />
+      {t(
+        'Avg: %s',
+        formatMetricUsingUnit(item.sum && item.count && item.sum / item.count, unit)
+      )}
+    </pre>
+  );
+}
+
+function Tags({
+  tags,
+  organization,
+  mri,
+}: {
+  mri: MRI;
+  organization: Organization;
+  tags: Record<string, string>;
+}) {
+  const [showingAll, setShowingAll] = useState(false);
+
+  const renderedTags = Object.entries(tags).slice(0, showingAll ? undefined : 5);
+  const renderText = showingAll ? t('Show less') : t('Show more') + '...';
+
+  return (
+    <StyledPills>
+      {renderedTags.map(([tagKey, tagValue]) => (
+        <Pill key={tagKey} name={tagKey}>
+          <Link
+            to={getDdmUrl(organization.slug, {
```

Comment:
I've noticed that when you click on the env tag with a value of prod, it adds it to the query filter instead of the project selector -> we should probably handle this case.

Reply:
to be fair, it is a valid query tag. Wouldn't it be confusing if clicking on different tags results in different ways of filtering 🤔

Reply:
Ah, no, you're right. I thought it was not working properly, but that's because we only have one tag value (prod) so applying it didn't change the shape of the chart at all. And I forgot we have env in group by and in autocomplete of the query search bar and everything.
ignore 👍